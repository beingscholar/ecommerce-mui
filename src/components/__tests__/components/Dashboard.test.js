import React from 'react'
import {render, cleanup} from '@testing-library/react'
import Dashboard from '../../Dashboard';

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Dashboard />)
    
    expect(asFragment(<Dashboard />)).toMatchSnapshot()
   })