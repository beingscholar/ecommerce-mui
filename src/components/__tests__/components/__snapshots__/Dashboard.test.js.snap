// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Dashboard
    </h1>
  </div>
</DocumentFragment>
`;
